pages/common/aws-s3-website.md

```diff
@@ -0,0 +1,12 @@
+# aws-s3-website
+
+> Set the website configuration for a bucket.
+> More information: <https://docs.aws.amazon.com/cli/latest/reference/s3/website.html>.
+
+- Configure a bucket as a static website:
+
+`aws s3 website s3://bucket-name/ --index-document index.html`
+
+- Configure an error page for the website:
+
+`aws s3 website s3://bucket-name/ --index-document index.html --error-document error.html`
```

Comment:
I have a minor suggestion for this page, we can wrap the placeholders inside `{{}}` to be highlighted in a different colour/style by our clients.

```suggestion
`aws s3 website {{s3://bucket-name/}} --index-document {{index.html}}`

- Configure an error page for the website:

`aws s3 website {{s3://bucket-name/}} --index-document {{index.html}} --error-document {{error.html}}`
```